### ISSUE TITLE
youtube backend

### ISSUE THREAD

**Issue by echavemann** [wry0313/gavin-db#4]
we should make use of the infinite free storage provided by YouTube to store files in the cloud

**Comment by stevenewald** [comment]
no, that is impractical, expensive, abusive, and overall unethical.

store it in [minecraft instead](url)
